### PR TITLE
add func errCodeToString

### PR DESCRIPTION
**Pull request by MOON-CLJ**
cc @everpcpc

### CODE REVIEW

**Comment by MOON-CLJ** [comment]
travis的setuptools有点问题，https://travis-ci.org/douban/libmc/jobs/385622435。

File: src/Client.cpp [hunk 1]
```diff
@@ -217,5 +217,31 @@ void Client::_sleep(uint32_t seconds) {
...
+    case RET_MC_SERVER_ERR:
+      return keywords::kSERVER_ERROR;
+    case RET_PROGRAMMING_ERR:
+      return keywords::kPROGRAMMING_ERROR;
+    case RET_INVALID_KEY_ERR:
+      return keywords::kINVALID_KEY_ERROR;
+    case RET_INCOMPLETE_BUFFER_ERR:
+      return keywords::kINCOMPLETE_BUFFER_ERROR;
+    default:
+      return "ok";
```
Comment (tclh123) [inline src/Client.cpp]: Add `case RET_OK:` and default return "unknown" ?

File: .travis.yml [hunk 2]
```diff
@@ -28,6 +28,8 @@ addons:
 
 install:
   - if [[ $TEST_SUITE != "cppcheck" ]]; then pip install --upgrade pip setuptools; fi
+  # FIXME tmp install twice, because travis.ci encounter https://github.com/pypa/setuptools/issues/523#issuecomment-337482959
+  - if [[ $TEST_SUITE != "cppcheck" ]]; then pip install --ignore-installed setuptools; fi
```
Comment (tclh123) [inline .travis.yml]: leave `.travis.yml` unchanged? not seeing a error like this now.

**Comment by MOON-CLJ** [comment]
@tclh123 updated

**Comment by tclh123** [comment]
There's a typo in commit messages
supress ->suppress

**Comment by MOON-CLJ** [comment]
> There's a typo in commit messages
> supress ->suppress

so strict? should i fix it ?

File: include/Client.h [hunk 3]
```diff
@@ -79,5 +79,7 @@ DECL_RETRIEVAL_CMD(gets)
   std::vector<unsigned_result_t*> m_outUnsignedResultPtrs;
 };
 
+const char* errCodeToString(err_code_t err);
```
Comment (tclh123) [inline include/Client.h]: maybe move errCodeToString to `Common.h` ?

File: src/c_client.cpp [hunk 4]
```diff
@@ -3,6 +3,7 @@
 
 
 using douban::mc::Client;
+using douban::mc::errCodeToString;
```
Comment (tclh123) [inline src/c_client.cpp]: are we need this?

**Comment by tclh123** [comment]
> > There's a typo in commit messages
> > supress ->suppress
> 
> so strict? should i fix it ?

It's quick to fix this.

File: src/c_client.cpp (hunk 4 above)
Comment (MOON-CLJ) [reply src/c_client.cpp]: @tclh123 rm this line seems also works。but do you know why?

File: include/Client.h (hunk 3 above)
Comment (MOON-CLJ) [reply include/Client.h]: okay

File: src/c_client.cpp (hunk 4 above)
Comment (MOON-CLJ) [reply src/c_client.cpp]: okay there is some "Client" usage in c_client.cpp。

**Comment by MOON-CLJ** [comment]
updated

**Pull request merged**